data/input.json

```diff
@@ -86,5 +86,22 @@
   "experimentalDeviation": {
     "description": "Experimental deviation (percent)",
     "value": 10
+  },
+  "randomness": {
+    "description": "the option for establishing new randon configuration or load from the file",
+    "value": 0 
+  },
+  "ageDistribution" : {
+    "description" : "distribution of age in pedestrians",
+    "distribution": {
+      "normal" : {
+        "description" : "normal distribution",
+        "samples" : 50,
```

Comment:
Just leave `samples` field as "numOfAgents", we don't wanna change it everytime we change the sample size. Just call for "numOfAgents" when you wanna use in code.

Reply:
Or make it `{"$ref": "#/numOfAgents/value"}`